string/aarch64/strncmp-mte.S

```diff
@@ -74,6 +70,7 @@ ENTRY_ALIGN (__strncmp_aarch64_mte, 0)
 	   (=> (X - 1) & ~(X | 0x7f)) is non-zero iff a byte is zero, and
 	   can be done in parallel across the entire word.  */
 	/* Start of performance-critical section  -- one 64B cache line.  */
+	.p2align 6
```

Comment:
This should be .p2align 4 - aligning to a cacheline is a complete waste.

Reply:
Done